README.md

```diff
@@ -123,6 +123,8 @@ If you want to redeploy to the same production URL you can run `yarn vercel --pr
 
 **Hint**: We recommend connecting the project GitHub repo to Vercel so you the gets automatically deployed when pushing to `main`
 
+**Hint**: If you don't have any contract deployed through SE-2 and plan to just deploy NextJS app maybe only with "connect wallet" functionality you can do so by just setting `targetNetwork : chains.mainnet` or your preferred network in `packages/nextjs/scaffold.config.ts`, rest all instruction remains same as [above](#deploying-your-nextjs-app)
```

Comment:
Added a small `hint` it can be improved a lot please feel free to suggest idea, aslo was wondering should we create a separate section ? I feel like its not worth it because README.md has really grown alot

Reply:
Update the hint text a bit to fit in properly, hope it makes sense : 
![Screenshot 2023-04-21 at 7 45 10 PM](https://user-images.githubusercontent.com/80153681/233659034-ac1a9790-f50f-406e-94e4-29b472929f71.jpg)

Before : 
![Screenshot 2023-04-21 at 7 26 36 PM](https://user-images.githubusercontent.com/80153681/233659219-41f2b4c4-910e-4810-8ffa-2e031c7d119d.jpg)

The empty space looked weird to me but happy to revert 🙌